src/test/java/spoon/test/comment/CommentTest.java

```diff
@@ -263,7 +264,12 @@ public void testInLineComment() {
 		assertEquals(createFakeComment(f, "comment after array value"), arrayValue.getComments().get(1));
 
 
-		CtReturn ctReturn = m1.getBody().getStatement(12);
+		CtLocalVariable ctLocalVariableString = m1.getBody().getStatement(12);
+		assertEquals(createFakeComment(f, "comment multi line string"), ((CtBinaryOperator)((CtBinaryOperator)ctLocalVariableString.getDefaultExpression()).getRightHandOperand()).getLeftHandOperand().getComments().get(0));
+		assertEquals("\"\" + (\"\"// comment multi line string" + newLine
+				+ " + \"\")", ctLocalVariableString.getDefaultExpression().toString());
+
+		CtReturn ctReturn = m1.getBody().getStatement(13);
 		assertEquals(createFakeComment(f, "comment return"), ctReturn.getComments().get(0));
```

Comment:
I'm not sure to understand the relation between your changes and the new test: you change the behaviour of comments for conditionnals, if and executables and you test a multiline string?... I think I miss something.